images/build/Dockerfiles/Dockerfile

```diff
@@ -197,30 +197,6 @@ FROM dotnet-install AS buildscriptbuilder
 
 FROM mcr.microsoft.com/oryx/base:build-yarn-cache-${YARN_CACHE_BASE_TAG} AS yarn-cache-base
 
-###
-# Build run script generators (to be used by the `oryx run-script` command)
```

Comment:
Avoid doing this change in this PR and handle it separately. This way you can delete this content as well as the corresponding command, tests etc. in the same PR. In case we want to revert a commit, we can hopefully.